internal/node/node.bzl

```diff
@@ -269,6 +269,19 @@ _NODEJS_EXECUTABLE_ATTRS = {
         allow_files = True,
         aspects = [sources_aspect, module_mappings_runtime_aspect, collect_node_modules_aspect],
     ),
+    "default_env_vars": attr.string_list(
+        doc = """Default environment variables that are added to `configuration_env_vars`.
+
+        This is separate from the default of `configuration_env_vars` so that a user can set `configuration_env_vars`
```

Comment:
I think the indent here is wrong and will make docs look wrong

Reply:
Right. There a mix everywhere. Will fix here. Should make a PR to just clean them up everywhere.